data/domain/world-2/votes.tsx

```diff
@@ -0,0 +1,101 @@
+        votes.bonuses.push(new VoteBonus(9,"Boosts all Choppin' EXP gain and Choppin' Efficiency by +{%",38,0));
+        votes.bonuses.push(new VoteBonus(10,"Boosts all Catching EXP gain and Catching Efficiency by +{%",46,0));
+        votes.bonuses.push(new VoteBonus(11,"Increases the amount of resources produced by the 3D Printer by }x",20,3));
+        votes.bonuses.push(new VoteBonus(12,"Boosts liquid generation rate for all Alchemy liquids by +{%",25,0));
+        votes.bonuses.push(new VoteBonus(13,"Boosts all Cooking EXP gain and Cooking Speed by +{%",63,4));
+        votes.bonuses.push(new VoteBonus(14,"Boosts Dungeon Credit and Dungeon Flurbo gain by }x",50,0));
+        votes.bonuses.push(new VoteBonus(15,"All your characters gain +{% more Class EXP from monsters",60,0));
+        votes.bonuses.push(new VoteBonus(16,"Speeds up Egg Incubation time and Breeding EXP gain by +{%",50,4));
+        votes.bonuses.push(new VoteBonus(17,"Boosts Sigil EXP gain by }x, still requires Sigils active in Lab",80,4));
+        votes.bonuses.push(new VoteBonus(18,"Boosts Construction Build Rate and Construction EXP gain by +{%",40,3));
+        votes.bonuses.push(new VoteBonus(19,"Boosts Shrine EXP gain by a staggering }x",53,3));
+        votes.bonuses.push(new VoteBonus(20,"Boosts Artifact Find chance in Sailing by }x",31,5));
+        votes.bonuses.push(new VoteBonus(21,"Boosts New Species chance when using DNA in Gaming by }x",80,5));
+        votes.bonuses.push(new VoteBonus(22,"Find +{% more Gold Nuggets when digging with the Shovel in Gaming",75,5));
+        votes.bonuses.push(new VoteBonus(23,"Boosts Divinity PTS gain by }x and Divinity EXP Gain by +{%",60,5));
+        votes.bonuses.push(new VoteBonus(24,"Boosts Sailing Captain EXP gain and Sailing Speed by }x",50,5));
+        votes.bonuses.push(new VoteBonus(25,"Boosts Sneaking Stealth by }x and EXP Gain by +{% for all your Ninja Twins",65,6));
+        votes.bonuses.push(new VoteBonus(26,"Boosts bonuses from all Golden Food by +{%",30,0));
+        votes.bonuses.push(new VoteBonus(27,"Increases Drop Rate for all your characters by +{%",38,0));
+        votes.bonuses.push(new VoteBonus(28,"Boosts Summoning EXP gain by +{% and all Essence gained by }x",40,6));
+        votes.bonuses.push(new VoteBonus(29,"Boosts Crop Value, and Farming EXP gain, AND Next Crop Chance by +{%",40,6));
+        votes.bonuses.push(new VoteBonus(30,"Increases Trapping EXP gain and Worship EXP gain by +{%",54,3));
+        votes.bonuses.push(new VoteBonus(31,"Increases Lab EXP gain by +{%",90,4));
+        votes.bonuses.push(new VoteBonus(32,"Boosts Equinox Bar Fill rate by }x",40,3));
+        votes.bonuses.push(new VoteBonus(33,"Boosts Refinery Cycle Speed by +{%",50,3));
+        votes.bonuses.push(new VoteBonus(34,"Increases cash earned from monsters by +{%",52,0));
+    }
+
+    // This one should be only used for display purpose, as it'll always return it's value even if bonus isn't active
```

Comment:
Why? 😆 

We often have `getBonusText` for display only purposes if you want to separate the behaviours.

But yeah overall just curious as to why always return the bonus? it's risky if we forget this is the case and rely on it later for some math.

Reply:
You're right, gonna make a few changes to avoid problems